utility/servicer/module_test.go

```diff
@@ -11,25 +15,53 @@ import (
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/modules"
 	mockModules "github.com/pokt-network/pocket/shared/modules/mocks"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
 )
 
-var testServicer1 = &coreTypes.Actor{
-	ActorType: coreTypes.ActorType_ACTOR_TYPE_SERVICER,
-	Address:   "a3d9ea9d9ad9c58bb96ec41340f83cb2cabb6496",
-	PublicKey: "a6cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d75a998b",
-	Chains:    []string{"0021"},
-}
+const (
+	testAppsTokensMultiplier = int(2)
+	testCurrentHeight        = int64(9)
+)
+
+var (
+	testServicer1 = &coreTypes.Actor{
+		ActorType: coreTypes.ActorType_ACTOR_TYPE_SERVICER,
+		Address:   "a3d9ea9d9ad9c58bb96ec41340f83cb2cabb6496",
+		PublicKey: "a6cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d75a998b",
```

Comment:
How did you generate these?

We have utilities to generate keys that are guaranteed to be compatible.

```go
	operatorKey, err := crypto.GeneratePublicKey()
	if err != nil {
		return nil, err
	}

	outputAddr, err := crypto.GenerateAddress()
	if err != nil {
		return nil, err
	}
```

Reply:
Thank you for pointing this out. Fixed.